homework3/semi-uber/UI/src/app/regestration-page/regestration-page.component.ts

```diff
@@ -40,4 +45,33 @@ export class RegestrationPageComponent implements OnInit {
   get username() {
     return this.userData.get("username");
   }
+
+  getRegForm() {
```

Comment:
`getRegForm` is not really appropriate name here. Cause what you are doing in this method is submitting the data from the from, so please consider about changing this name.

Reply:
registrateUser - renamed